src/firefly/java/edu/caltech/ipac/firefly/tools/FitsHeaderTool.java

```diff
@@ -63,17 +59,6 @@ public static void main(String[] args) throws FitsException, IOException {
         }
     }
 
-    private static void initLog4j() {
-        Properties prop = new Properties();
-        prop.setProperty("log4j.appender.terminal", "org.apache.log4j.ConsoleAppender");
-        prop.setProperty("log4j.appender.terminal.layout", "org.apache.log4j.PatternLayout");
-        //prop.setProperty("log4j.appender.terminal.layout.ConversionPattern","%d{MM/dd HH:mm:ss} %5p - %.2000m [%t] (%c{2}#%M:%L)%n");
-        prop.setProperty("log4j.logger.edu.caltech.ipac", "INFO, terminal");
-//        prop.setProperty("log4j.additivity.edu.caltech.ipac","true");
-//        PropertyConfigurator.configure("/tmp/log4j.properties");
-        PropertyConfigurator.configure(prop);
-    }
-
```

Comment:
@loitly this utility class was giving the result output in the console using Log4j - if this part needed to be removed, how/where did you replaced it? Thanks!

Reply:
Ah.  I see now.  I wasn't sure what this is about.  Ok.  Let me suggest a solution.
Log4J will startup with a default configuration if one is not given.  It's set to console with a level of ERROR.
I suggest when writing code, to log at different LEVEL, then set a default level you want it to normally output.  i.e.
` Logger.setProLog(Level.INFO, null) `
Then, if you want to see lower levels, like `debug` or `trace`, add this to the command line.
`-Dorg.apache.logging.log4j.level=trace`
This is a clean programmatic way of controlling logs.